.github/workflows/push.yaml

```diff
@@ -108,9 +106,9 @@ jobs:
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_TOKEN }}
 
-      - name: Build and push images for ${{ matrix.scope }}
+      - name: Build and push images
         env:
           PR_SHA: ${{ github.event.pull_request.head.sha }}
         run: |
           export TAG=pr-${{ github.event.pull_request.number }}-${PR_SHA:0:8}
-          IMAGE_BUILDER=docker hack/toolchain_build_push_${{ matrix.scope }}.sh "${TAG}"
+          IMAGE_BUILDER=docker ci/toolchain_build_push.sh "${TAG}"
```

Comment:
shouldn't it be `ci/toolchain_manager.sh`?

Reply:
Fixed in 6022e5a.